### PR TITLE
Password re-use checking and update credits doc for reuse

### PR DESCRIPTION
**Pull request by ghexp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,12 +49,17 @@ The JSON InSpec results file may also be loaded into a __full heimdall server__,
 To report a bug or feature request, please open an [issue](https://github.com/mitre/canonical-ubuntu-18.04-lts-stig-baseline/issues/new).
 
 ## Authors
-- Aaron Lippold
-- The MITRE InSpec Team
+- [Aaron Lippold](https://github.com/aaronlippold)
```
Comment (aaronlippold) [inline README.md:52]: Please update the authors to include yourself, and feel free to move me to the special thanks or reviewer section.

File: controls/V-219164.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # encoding: UTF-8
+min_fail_delay = input('min_fail_delay', value: 4000000,
```
Comment (aaronlippold) [inline controls/V-219164.rb:2]: There is no need to override the shared input from the inxpec.yml.

you can just call the input in the control `input('min_fail_delay')` and then set its default value in the `inspec.yml` in the `inputs:` section.

This allows us to centerlize the inputs and if a change is needed we can update it in one place

This also allows us to easily do a global find and replace in the controls should we want to change an inputs name.

File: controls/V-219180.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 # encoding: UTF-8
+min_reuse_generations = input('min_reuse_generations', value: 5,
```
Comment (aaronlippold) [inline controls/V-219180.rb:2]: same as above, use the `inspec.yml inputs:` section for this, and then call the input directly in the control.

File: controls/V-219164.rb (hunk 2 above)
Comment (aaronlippold) [reply controls/V-219164.rb:2]: if you have done this in other areas of the code and I missed it, please update it.

The only time I would use a local variable inside of a control is when I need to construct something from inputs or ask a question to the system.

**Review by aaronlippold — CHANGES_REQUESTED** [review]

File: controls/V-219164.rb (hunk 2 above)
Comment (ghexp) [reply controls/V-219164.rb]: Fixed these two.  I think there's a couple more to address--came from SIMP examples. I'll fix those under separate PR

Comment (ghexp) [reply controls/V-219164.rb]: A couple commits to do it right.  Not sure how they're queued by actions, but ignore the failures until the last push updates the PR.

**Comment by aaronlippold** [comment]
There's seems to be an issue in the vanilla roles vars file. Did it get removed be accident. It is causing the cicd to fall

**Comment by aaronlippold** [comment]
If I'm reading the weird in the actions output on the kitchen test step

**Comment by ghexp** [comment]
Avoiding the variables and just using input instead worked better.  Seemed to somehow impact vanilla though I don't know how.

**Comment by ghexp** [comment]
I'll merge the other inspec variable input change once this PR is merged

**Pull request merged**